vm/devices/storage/nvme/src/pci.rs

```diff
@@ -484,17 +481,22 @@ impl PciConfigSpace for NvmeController {
     }
 }
 
-impl SaveRestore for NvmeController {
-    type SavedState = SavedStateNotSupported;
+mod save_restore {
```

Comment:
Why is this changing?

Reply:
I am sure that one of the unit tests was failing, but I cannot reproduce it anymore, will revert back.